src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -1309,3 +1320,38 @@ def visit_SymRef(self, node: gtir.SymRef) -> IteratorExpr | MemletExpr | SymbolE
         # if not in the lambda symbol map, this must be a symref to a builtin function
         assert param in gtir_python_codegen.MATH_BUILTINS_MAPPING
         return SymbolExpr(param, dace.string)
+
+    def apply(
+        self,
+        node: gtir.Lambda,
+        args: Sequence[IteratorExpr | MemletExpr | SymbolExpr],
+    ) -> tuple[list[DataflowInputEdge], DataflowOutputEdge]:
+        # lambda arguments are mapped to symbols defined in lambda scope
+        prev_symbols: dict[
+            str,
+            Optional[IteratorExpr | MemletExpr | SymbolExpr],
```

Comment:
Why do you use `None` to indicate not present (I guess). I would think it is much more logical they would simply be not present.

Reply:
I changed based on the suggestion in next comment.